### PR TITLE
added IBM SRE blogs

### PR DESCRIPTION
**Pull request by miss-stuck-a-lot**

### CODE REVIEW

**Comment by upgundecha** [comment]
Hi @miss-stuck-a-lot thanks a lot for submitting PR. Could you please fix following markdown lint error and resubmit?

```
README.md: 615: MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "*[AIOps tools and solutions](h..."]
```

File: README.md [hunk 1]
```diff
@@ -606,6 +606,15 @@ _Note to readers: This list refers to some of the articles, posts, videos, tools
...
 </details>
 
+<details>
+  <summary>IBM</summary>
+
+### Blog Posts
+
+* [What is Site Reliability Engineering (SRE)?](https://www.ibm.com/cloud/learn/site-reliability-engineering)
+* [AIOps tools and solutions](https://www.ibm.com/cloud/aiops)
+</details>
```
Comment (upgundecha) [inline README.md:616]: Add a new line before this

Comment (miss-stuck-a-lot) [reply README.md:617]: thank you so much

**Comment by miss-stuck-a-lot** [comment]
thank you for your patience, this is my first time fixing errors on my commit, and also open source

**Comment by upgundecha** [comment]
Awesome, thanks @miss-stuck-a-lot 🙌

**Pull request merged**